frontend/pages/index.tsx

```diff
@@ -39,7 +41,9 @@ export default function HomePage() {
   if (!isWeb3Enabled) return <Login />;
   return (
     <div style={{ backgroundColor: "rgba(232, 249, 252, 0.76)" }}>
+      <Loading/>
       <Container className="mt-5 py-3 bg-white">
+      <Suspense fallback={<div>Loading...</div>}>
```

Comment:
Instead of this can you directly put Loading inside Suspense? 
like, `<Suspense fallback={<Loading/>}>`